medicines/pars-upload/cypress/integration/tests.js

```diff
@@ -63,23 +63,23 @@ describe('PARs upload form', () => {
   it('can add multiple substances', () => {
     cy.visit('/new-par')
 
-    cy.findByLabelText('Product name').type('Ibuprofen pills')
+    cy.findByLabelText('Brand/Generic name').type('Ibuprofen pills')
 
     cy.findByLabelText('Strength').type('Really powerful stuff')
 
     cy.findByLabelText('Pharmaceutical dose form').type('some form')
 
-    cy.findByLabelText('Active substance').type('Ibuprofen')
+    cy.findByLabelText('Active substance(s)').type('Ibuprofen')
 
-    cy.findByText('Add another active substance').click()
+    cy.findByText('Add another Active substance(s)').click()
```

Comment:
maybe don't do _Find and replace all_

Reply:
`Add another Active substance(s)` feels grammatically wrong?
how about
`Add another Active substance`
or
`Add more Active substances`

also is the capitalisation of Active correct?  lowercase looks better to me.

Reply:
ah just realised to was the "replace all" you mention @rjkerrison, yes, agree.